postman/sanity.postman_collection.json

```diff
@@ -0,0 +1,89 @@
+{
+	"info": {
+		"_postman_id": "4a4deeb8-dae7-4bdc-98ba-c457b1bffff1",
+		"name": "sanity",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "Submit Claim",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+						"mode": "formdata",
```

Comment:
The submit API isn't formdata, should be sent as-is in the body with header `application/cose`.